vnext/Microsoft.ReactNative/Views/ViewViewManager.cpp

```diff
@@ -345,10 +345,7 @@ folly::dynamic ViewViewManager::GetNativeProps() const {
   auto props = Super::GetNativeProps();
 
   props.update(folly::dynamic::object("pointerEvents", "string")("onClick", "function")("onMouseEnter", "function")(
-      "onMouseLeave", "function")
-               //("onMouseMove", "function")
```

Comment:
This has been commented out, but mostly implemented in Native for multiple years apparently. Not exposed to the JS at all. I think I'm going to remove the dead code in a future PR, since we're wanting to be more intentional about mouse events.